data_model/src/asset.rs

```diff
@@ -287,22 +285,8 @@ impl AssetDefinition {
     /// Construct builder for [`AssetDefinition`] identifiable by [`Id`].
     #[must_use]
     #[inline]
-    pub fn quantity(id: AssetDefinitionId) -> <Self as Registered>::With {
-        <Self as Registered>::With::new(id, AssetValueType::Quantity)
-    }
-
-    /// Construct builder for [`AssetDefinition`] identifiable by [`Id`].
-    #[must_use]
-    #[inline]
-    pub fn big_quantity(id: AssetDefinitionId) -> <Self as Registered>::With {
-        <Self as Registered>::With::new(id, AssetValueType::BigQuantity)
-    }
-
-    /// Construct builder for [`AssetDefinition`] identifiable by [`Id`].
-    #[must_use]
-    #[inline]
-    pub fn fixed(id: AssetDefinitionId) -> <Self as Registered>::With {
-        <Self as Registered>::With::new(id, AssetValueType::Fixed)
+    pub fn numeric(id: AssetDefinitionId) -> <Self as Registered>::With {
+        <Self as Registered>::With::new(id, AssetValueType::Numeric(NumericSpec::default()))
```

Comment:
I must say I don't get this now. How can the user define a numeric asset with non-default `NumericSpec`?

Reply:
by using `AssetDefinition::new(definition_id, AssetValueType::Numeric(NumericSpec::integer()))`

Reply:
ok